backend/src/api/middlewares/validation.ts

```diff
@@ -1,13 +1,12 @@
 import { NextFunction, Request, Response } from "express";
 import { getLogger } from "../../utils/logger";
-import Joi from "joi";
 
 const logger = getLogger();
 
 const validationMiddleware =
-  (schema: Joi.ObjectSchema, property: "body" | "query") =>
+  (schema: any, property: "body" | "query") =>
```

Comment:
Is there a reason why we're defining the schema as `any` here?

Reply:
According to the documentation, there is no equivalent to Joi.ObjectSchema in Zod 😔. So I figured it was a good call since the schema will be in "any" object types anyway 👀.

Reply:
Fair enough